src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
+const arr = [];
+const texts = [...population].map(item => item.innerText);
```

Comment:
why do we need copy array for map method
```suggestion
const texts = population.map(item => item.innerText);
```

Reply:
We don't need copy of array, but without spread operator map doesn't work, as variable population isn't array, it's collection. So I transformed it into the array with spread operator, as it was shown in the video.